### DISCUSSION TITLE
Screen style menu bug

### DISCUSSION THREAD

**Discussion by Cnxtrys** [Open-Shell/Open-Shell-Menu#1839] [General]
When i open up shell menu I can only go into certain settings and I cant go into settings such as start menu style. I am confused because I was just able to edit this yesterday not it is not available. please help me sum1

**Comment by Cnxtrys** [comment]
also, its just blank.

**Reply by ge0rdi** [reply]
What is blank exactly?
Could you post screenshot for clarification?

**Comment by CTVCAM8** [comment]
I think you just need to tick the box named "Show all settings"

![Show all settings](https://github.com/Open-Shell/Open-Shell-Menu/assets/44127216/1398b333-fb69-48ca-8b62-0459da0f687e)
